lib/config/default-scopes.js

```diff
@@ -2,16 +2,17 @@
 
 export default [
   {
-    scopes: ['source.js', 'source.js.jsx', 'source.coffee', 'source.coffee.jsx', 'source.ts', 'source.tsx'],
+    scopes: ['source.js', 'source.js.jsx', 'source.coffee', 'source.coffee.jsx', 'source.ts', 'source.tsx', 'text.html.vue'],
     prefixes: [
       'import\\s+.*?from\\s+[\'"]', // import foo from './foo'
       'require\\([\'"]', // require('./foo')
       'define\\(\\[?[\'"]' // define(['./foo']) or define('./foo')
     ],
-    extensions: ['js', 'jsx', 'ts', 'tsx', 'coffee'],
+    extensions: ['js', 'jsx', 'vue', 'ts', 'tsx', 'coffee'],
     relative: true,
     replaceOnInsert: [
       ['.jsx?$', ''],
+      ['.vue$', ''],
```

Comment:
this line should be removed!
it's better keep the .vue extension instead to remove it !!!